lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDP.h

```diff
@@ -90,7 +90,7 @@ class ProcessKDP : public lldb_private::Process {
   // Process Control
   lldb_private::Status WillResume() override;
 
-  lldb_private::Status DoResume() override;
+  lldb_private::Status DoResume(lldb::RunDirection direction) override;
```

Comment:
We should use `DoResumeReverse()` in Process.h and this change won't need to happen.

Reply:
The decision was to use a direction flag.